pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
```

Comment:
I know this is the original description, but the goal of the sprint is to make it better. After reading these texts, it's still difficult for me to understand what this method is about. In which cases it'll be useful for me.